.github/workflows/tests.yml

```diff
@@ -101,41 +101,70 @@ jobs:
           npm install -g nodemon
           npm init -y
           npm install --force
-          npm test -- --coverage || true
```

Comment:
Why are we remove this flag? 
Also should this be npm run test? Like we have for the web_app_server?

Reply:
It is already in the package.json file